.ci/updatecli/updatecli-bump-stack-version.yml

```diff
@@ -0,0 +1,51 @@
+# update-cli configuration for automated go updates
```

Comment:
Shall we change this with a more descriptive details? Maybe it's not needed since `name` is descriptive enough. WDYT?

Reply:
```suggestion
```